DIMS-Core/DIMS-Core.DataAccessLayer/Repositories/TaskStateRepository.cs

```diff
@@ -1,16 +1,30 @@
 using DIMS_Core.DataAccessLayer.Models;
+using Microsoft.EntityFrameworkCore;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
-using System.Threading.Tasks;
+using AsyncTask = System.Threading.Tasks.Task;
 
 namespace DIMS_Core.DataAccessLayer.Repositories
 {
     public class TaskStateRepository : Repository<TaskState>
     {
         public TaskStateRepository(DIMSContext context) : base(context)
         {
+            Context = context;
+        }
+
+        public DIMSContext Context { get; }
+
+        public async AsyncTask SetUserTaskAsFail(int userId, int taskId)
+        {
+            await Context.Database.ExecuteSqlRawAsync("execute SetUserTaskAsFail @UserId, @TaskId", userId, taskId);
+        }
+
+        public async AsyncTask SetUserTaskAsSuccess(int userId, int taskId)
+        {
+            await Context.Database.ExecuteSqlRawAsync("execute SetUserTaskAsSuccess @UserId, @TaskId", userId, taskId);
```

Comment:
It is better to move these methods to UserTaskRepository.

Reply:
@VladPulyak Also you will have too add interface for this repository with these methods

Reply:
Sorry. I didn't notice file name

Reply:
Changed